gitea/user_app.go

```diff
@@ -40,6 +41,17 @@ func (c *Client) ListAccessTokens(user, pass string) ([]*AccessToken, error) {
 // swagger:parameters userCreateToken
 type CreateAccessTokenOption struct {
 	Name string `json:"name" binding:"Required"`
+	MatchOwner []string `json:"match_owner,omitempty"`
+	MatchRepo []string `json:"match_repo,omitempty"`
+	RegexMatchBranch []string `json:"regex_match_branch,omitempty"`
```

Comment:
Although regex is more powerful for ease of use I would prefer wildcard match:
https://golang.org/pkg/path/filepath/#Match

Reply:
thanks and agree. regex was the word came to my mind but actually I meant wildcard.